ir/type.def

```diff
@@ -233,6 +233,14 @@ abstract Type_StructLike : Type_Declaration, ISimpleNamespace, IAnnotated {
         return fields.getDeclarations(); }
     StructField getField(cstring name) const {
         return fields.getDeclaration<StructField>(name); }
+    int getOffset(cstring name) const {
```

Comment:
Please change this function name to something more explicit, such as getFieldIndex.